### ISSUE TITLE
Migrated question: Roomba forecaster forecast deletion, it will not

### ISSUE THREAD

**Issue by braiam** [makyen/StackExchange-userscripts#1]
In [this question](http://softwareengineering.stackexchange.com/q/122569/104338) it forecast that the closedabandoned will delete this question, but the question has positive score and answers with positive score. Not sure if the API is lying as I couldn't find any fault with the logic used.

**Comment by makyen** [comment]
Thanks for reporting the issue.

It's nearly certain that question will actually be deleted. Certainly not after 5 years of being closed. 

However, it does meet all the stated criteria for the [`RemoveRejectedMigrations` Roomba](http://stackoverflow.com/help/roomba).  The stated criteria for that Roomba task are that the question was migrated to the current site and then rejected. Rejected is defined in [this meta post](https://meta.stackexchange.com/a/10250/271271) as being closed (for other than being a duplicate), or deleted.  Clearly, there is at least one more criteria that has to be met, which is not met by that question.

My guess is that this question is not being deleted because it is grandfathered out of the process. It could be that there is some date for which if either the migration or the close are before that date, then the Roomba does not delete. Another possibility is that the method of migrating was different between then and now. For example, the above meta post states that there are employee performed migrations, which I hypothesize are not subject to this Roomba task.

Given that the criteria are unstated, I've handled it by checking if enough time has passed from when the question was closed for the weekly Roomba to have had two chances to delete the question. If it is not deleted after that, then it is assumed that it will not be deleted due to either of the Roomba migration tasks.

Note: I had missed that the must not be closed as a duplicate. So, the code has been changed fix that issue also, but it was not affecting the question you linked.
